app/models/folio/holding_location.rb

```diff
@@ -0,0 +1,13 @@
+module Folio
+  class HoldingLocation
+    def initialize(effective_location:)
+      @effective_location = effective_location
+    end
+
+    def self.from_dynamic(json)
+      new(effective_location: Location.from_dynamic(json.fetch('effectiveLocation')))
+    end
+
+    attr_reader :effective_location
+  end
+end
```

Comment:
That structure exists, but if we're not using it, it would just add bloat without offering anything.

Reply:
since we know for sure we need effective location, why not just make `effective_location` a `Folio::Location` on `Folio::Holding` and add the rest later if we need them?

Reply:
I wanted to follow the json structure, like `holding.location.effective_location`, which is what this does.  We can add `holding.location.permanent_location`, etc later if needed.

Reply:
isn't the JSON structure just an artifact of how we wrote the postgres query? I'm saying `HoldingLocation` will only ever have `Location`s as properties; there's no unique logic or values that justify it having a class. Let's just put the `Location` directly on `Holding`. I don't think there will ever be data associated with the relationship itself, right?

Reply:
we confirmed the RAMLs don't have a top-level `location` key; I'm gonna change the postgres query to also flatten out the various permanent/temporary/effective location keys.